Dockerfile

```diff
@@ -17,10 +17,10 @@ ARG DBNAME="data"
 ARG PORT=3000
 ARG CLUSTER_PORT=4000
 ARG HOST="0.0.0.0"
-ARG PATH="."
+ARG DBPATH="."
```

Comment:
Couple things on this:

- `PATH` is an important environment variable on shells for finding resources, so jumping into the docker container to debug with PATH set to the /app directory means no commands (`ls`, `cat`, etc) will be found
- `start.js` ends up asking the environment for a `DBPATH` anyways, so `PATH` simply doesn't match up to what's expected from the other side